local_groups/urls.py

```diff
@@ -16,74 +16,74 @@
-                ),
-            ])),
+# TODO: move to organizing hub app
+urlpatterns += [
+    url(r'^organizing-hub/', include([
+        url(r'^event/', include([
             url(
-                r'^login/',
-                auth_views.login,
-                {
-                    'authentication_form': GroupLoginForm,
-                    'redirect_authenticated_user': True
-                },
-                name='groups-login'
+                r'^create/',
+                EventCreateView.as_view(),
+                name='groups-event-create'
             ),
+        ])),
+        url(
+            r'^login/',
+            auth_views.login,
+            {
+                'authentication_form': GroupLoginForm,
+                'redirect_authenticated_user': True
+            },
+            name='groups-login'
+        ),
+        url(
+            r'^logout/',
+            auth_views.logout,
+            {'next_page': 'groups-login'},
+            name='groups-logout'
+        ),
+        url(r'^password/', include([
             url(
-                r'^logout/',
-                auth_views.logout,
-                {'next_page': 'groups-login'},
-                name='groups-logout'
+                r'^change/',
+                GroupPasswordChangeView.as_view(),
+                name='groups-password-change',
             ),
-            url(r'^password/', include([
+            url(r'^reset/', include([
                 url(
-                    r'^change/',
-                    GroupPasswordChangeView.as_view(),
-                    name='groups-password-change',
+                    r'^$',
+                    auth_views.password_reset,
+                    {'password_reset_form': GroupPasswordResetRequestForm},
+                    name='password_reset',
                 ),
-                url(r'^reset/', include([
-                    url(
-                        r'^$',
-                        auth_views.password_reset,
-                        {'password_reset_form': GroupPasswordResetRequestForm},
-                        name='password_reset',
-                    ),
-                    url(
-                        r'^confirm/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
-                        GroupPasswordResetView.as_view(),
-                        name='password_reset_confirm',
-                    ),
-                    url(
-                        r'^done/',
-                        auth_views.password_reset_done,
-                        name='password_reset_done',
-                    ),
-                ])),
-            ])),
-            url(r'^verify-email/', include([
                 url(
-                    r"^confirm/(?P<key>[-:\w]+)/$",
-                    VerifyEmailConfirmView.as_view(),
-                    name="groups-verify-email-confirm"
+                    r'^confirm/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
```

Comment:
What's happening here? Wasn't this just a config change?

Reply:
This should all just be an indentation change. Github is no bueno at displaying trivial indent changes 😞

Reply:
Oh Okay I see now, thanks for clarifying!